src/language-markdown/printer-markdown.js

```diff
@@ -61,14 +63,13 @@ function genericPrint(path, options, print) {
       options.originalText.slice(
         node.position.start.offset,
         node.position.end.offset
-      ),
-      options
+      )
     ).map((node) =>
       node.type === "word"
         ? node.value
         : node.value === ""
         ? ""
-        : printLine(path, node.value, options)
+        : printWhitespace(path, node.value, options)
```

Comment:
I wonder if it's okay that `printWhitespace` is called without the 4th argument here.

Reply:
I didn't know what I should designate. Now I found one.